lib/archethic/bootstrap.ex

```diff
@@ -207,13 +209,83 @@ defmodule Archethic.Bootstrap do
       Logger.info("Synchronization finished")
     end
 
+    resync_network_chain([:oracle, :node_shared_secrets], P2P.authorized_and_available_nodes())
+
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
 
     :persistent_term.put(:archethic_up, :up)
     Listener.listen()
   end
 
+  # when no node loaded in memtable
+  @spec resync_network_chain(any, maybe_improper_list) :: :ok
+  def resync_network_chain(_type_list, _nodes = []),
+    do: Logger.info("Enforced Resync of Network Txs: failure, No-Nodes")
+
+  def resync_network_chain(type_list, nodes) when is_list(nodes) do
+    # spawn process for each type of txn to sync
+    Task.Supervisor.async_stream_nolink(Archethic.TaskSupervisor, type_list, fn type ->
+      # acquire a(any) address of type txn, (for querying DB)
+      case TransactionChain.list_addresses_by_type(type) |> Stream.take(1) |> Enum.at(0) do
+        nil ->
+          Logger.info("Enforced Resync: failure ", transaction_type: type)
```

Comment:
Why do you qualify it as failure ? If it's pre self repair any new node would have this log